packages/terra-form-select/CHANGELOG.md

```diff
@@ -3,7 +3,10 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed Dropdown closing issue on Hold and Scroll in IE
 
+------------------
```

Comment:
This divider can be removed, it adds an extra border to the generated markdown file

Reply:
divider has been removed.